.eslintrc.js

```diff
@@ -29,25 +28,50 @@ module.exports = {
         'testem.js',
         'blueprints/**/*.js',
         'config/**/*.js',
-        'tests/dummy/config/**/*.js'
+        'tests/dummy/config/**/*.js',
       ],
       excludedFiles: [
         'addon/**',
         'addon-test-support/**',
         'app/**',
-        'tests/dummy/app/**'
+        'tests/dummy/app/**',
       ],
       parserOptions: {
-        sourceType: 'script'
+        sourceType: 'script',
       },
       env: {
         browser: false,
-        node: true
+        node: true,
       },
-      plugins: ['node'],
-      rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
+      plugins: ['node', 'prettier'],
+      extends: ['plugin:node/recommended', 'prettier'],
+      rules: {
         // add your custom rules and overrides for node files here
-      })
-    }
-  ]
+      },
+    },
+    {
+      files: ['**/*.ts'],
+      parser: '@typescript-eslint/parser',
+      parserOptions: {
+        // allows eslint from any dir
+        tsconfigRootDir: __dirname,
+        project: ['./tsconfig.json'],
+      },
+      plugins: ['@typescript-eslint', 'ember', 'prettier'],
```

Comment:
This merges with the top level set, you should not need to specify `prettier` or `ember` again here.

Reply:
I’ll double check it. I was not seeing it work correctly, but it may have been that fixing it at top level *did* fix and I just failed to run in between!

Reply:
KK, I'm definitely interested in the experiment, lemme know if it doesn't work like I was thinking above...

Reply:
Seems like I just had too many operations in flight; this works as expected!